### ISSUE TITLE
Problem with fail2ban after a reboot

### ISSUE THREAD

**Issue by MX10-AC2N** [YunoHost-Apps/vaultwarden_ynh#154]
Hi all,
And first of all, thank you for porting Vaultwarden to Yunohost.
I am a new Vaultwarden user, I installed without any particular problems, I was able to import my password backup (export from bitwarden), in short, super happy ..
Last night I shut down my server, and this morning at startup I have an error message in the fail2ban service, here is the copy : https://paste.yunohost.org/utomolulet
The problem seems to be related to the fact that fail2ban cannot find the log of Vaultwarden, I check in / var / log is indeed no vaultwarden directory .. It's strange isn't it?
All other services work fine, here is the copy of the VaultWarden log :
https://paste.yunohost.org/emehusosuk
I specify that I also use Jellyfin on my Yunohost and that port 8096 and maybe already used, locally I connect to Jellyfin on port 8095 ...
I try reboot but the problem still here..
In short, I am a little lost to repair all this ..
Thanks for your help..

**Comment by MX10-AC2N** [comment]
It seems that I managed to fix the problem
by creating the "vaultwarden" directory and creating a "vaultwarden.log" file
`mkdir /var/log/vaultwarden
touch /var/log/vaultwarden.log`
After that I was able to restart the fail2ban service without error.
Perhaps it would be necessary to add at the end of the installation of Vaultwarden the creation of the first log file in order to avoid this kind of inconvenience.
I leave the problem open while it is seen by the devs, but then it can be considered "solved"
Thank you

**Comment by MX10-AC2N** [comment]
Hmmm, I did all my handling while logged in as "root" so my directory and the log file are "root:root" shouldn't I modify that?

**Comment by MX10-AC2N** [comment]
Hi,
Well finally my little manipulation to manually create the directory and file /var/log/vaultwarden/vaultwarden.log only works until restart.
After a restart of the server the directory and the log file have disappeared again and so I have fail2ban which bug again ..
A little help would be welcome.
Thanks in advance..
have a good day.

**Comment by MX10-AC2N** [comment]
It's good, problem solved, 
I had to create the directory and the .log file and the rights to /var/log/vaultwarden must be vaultwarden:vaultwarden and for /var/log/vaultwarden/vaultwarden.log vaultwarden:root
